### PR TITLE
Refreshed tensorflow datasets to the latest release

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,23 +24,10 @@ env:
 
 matrix:
   include:
-    - env: CONFIG=linux_ppc64le_python3.6 UPLOAD_PACKAGES=True DOCKER_IMAGE=ibmcom/powerai:1.6.2-pytorch-ubuntu18.04-py36-ppc64le
+    - env: CONFIG=linux_noarch UPLOAD_PACKAGES=True DOCKER_IMAGE=conda-forge/linux-anvil-ppc64le
```
Comment (wdirons) [inline .travis.yml]: `conda-forge` should be `condaforge`. That is the cause of the travis failure.

Comment (npanpaliya) [reply .travis.yml]: Thanks Billy! Sometimes, I think copy paste is better :)

File: conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
+# (C) Copyright IBM Corp. 2018, 2020. All Rights Reserved.
```
Comment (wdirons) [inline conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh]: Technically you only need to update copyrights for files you are modifying.

Comment (npanpaliya) [reply conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh]: True.But the PR is merged now. I'll change this in next update.

Comment (wdirons) [reply conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh]: There is no need to change it now.

**Pull request merged**